packages/btcindexer/src/models.ts

```diff
@@ -118,3 +118,45 @@ export interface PostNbtcTxRequest {
 export type SuiTxDigest = string;
 
 export type { BlockQueueRecord };
+
+export interface NbtcTxInsertion {
+	txId: string;
+	vout: number;
+	blockHash: string;
+	blockHeight: number;
+	suiRecipient: string;
+	amountSats: number;
+	nbtc_pkg: string;
```

Comment:
Maybe we should keep the same convention for the naming?

```suggestion
	nbtcPkg: string;
```